tools/cldr-apps/js/src/esm/cldrProgress.js

```diff
@@ -44,6 +44,7 @@ class MeterData {
     this.votes = votes || 0;
     this.total = total || 0;
     this.level = level || cldrText.get("coverage_unknown");
+    this.exceptional = false;
```

Comment:
would 'inactive' be a better term here?

Reply:
maybe.... I was starting to implement it with "status=exception" (per https://ant.design/components/progress/ ) and that's what inspired me to call it "exceptional", but then I ended up using "showInfo=false" (also  per https://ant.design/components/progress/  )